Cargo.toml

```diff
@@ -52,13 +53,18 @@ yaml-rust = "0.4.5"
 
 [dev-dependencies]
 criterion = "0.4.0"
-gix-testtools = "0.11.0"
 insta = { version = "1.29.0", features = ["json", "redactions"] }
+onefetch = { path = ".", features = ["test-utils"] }
```

Comment:
I don't think is line is needed, the `required-features` field you added in [[bench]] and [[test]] tables should be enough